documentation/docs/libraries/python/getting_started.md

```diff
@@ -28,26 +31,30 @@ You can download the latest artifacts for major python version using the  [nighl
 pip install <wheel_file>
 ```
 
-Once it has been properly installed you can double-check the installation using _pip_ :
+Once it has been installed, you can double-check the installation using _pip_ :
+
 ```bash
 pip list
 ```
 
 The pip list should now include the _iota-wallet-python-binding_:
+
 ```plaintext
 Package                    Version
 -------------------------- -------
 iota-wallet-python-binding 0.1.0
 ```
 
-Once you python environment has installed the `iota-wallet-python-binding` you can start developing using the python binding.
+Once your python environment has installed the `iota-wallet-python-binding` , you can start developing using the python binding.
```

Comment:
Shouldn't it be
```suggestion
Once your python environment has installed the `iota-wallet-python-binding`, you can start developing using the python binding.
```
?

Reply:
Sorry, VS+grammarly makes it hard with the spacing; never 100% sure. Fixing now👌